### PR TITLE
Qa components

### PR DESCRIPTION
**Pull request by kimhonrada**
Create qa component files
https://trello.com/c/1bIz0EXv/88-set-up-component-files-and-css-grid-for-qa-widget

Create container for QA List
https://trello.com/c/3ICmmjEE/69-create-container-component-skeleton-to-house-and-organize-blocks-of-qa-entries

Create individual QA entries, question and answer components
https://trello.com/c/PTbYirWM/66-create-container-component-skeleton-to-house-and-organize-individual-question-and-answer-containers
https://trello.com/c/lJtnlB45/68-create-container-component-skeleton-for-individual-answers
https://trello.com/c/mHK6LuHY/67-create-container-component-skeleton-for-individual-questions

### CODE REVIEW

**Comment by kimhonrada** [comment]
Please review, and leave open. Thank you!

**Review by ZloNAGit — APPROVED** [review]
Code looks nice. Interested to see how it's rendering in your sections. 👍

File: client/components/QandA/QAComponents/AnswerEntry.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const AnswerEntry = ({ answer }) => {
+  // line 5, prints single answer
+ // console.log('this is answers ', answer[0][1])
+  if (!answer[0]) {
+    return null;
+  } else {
+    return (
+      <div id="AnswerEntry">
+        <div>
+        Answer: {answer[0][1].body}
```
Comment (Jplier) [inline client/components/QandA/QAComponents/AnswerEntry.jsx:12]: For the sake of conciseness maybe assign this to a variable. like answer = answer[0][1]

File: dist/styles.css [hunk 2]
```diff
@@ -27,6 +27,13 @@
   margin: 10px;
 }
 
+#QAList {
+  grid-row-start: 2;
```
Comment (Jplier) [inline dist/styles.css:31]: for shorthand you can do grid-row: 2 / 3;

**Review by Jplier — COMMENTED** [review]
Looks good nice job! Just a few things to make the code more clean but no problems that i see. I like your use of destructuring, i will try to incorporate that more as well.

**Pull request merged**